recipe/meta.yaml

```diff
@@ -51,7 +51,7 @@ outputs:
         - python >=3.7
       run:
         - jupyter-lsp >={{ server_version }}
-        - jupyterlab >=3,<4
```

Comment:
The 3.x branch is still tested against 3.1 on CI and tests are passing so I would relax the pin range if that's not a problem

Reply:
Unless you need that for licences stuff?

Reply:
no, 3.1 added the license stuff, and i see that's the [actual current pin](https://github.com/jupyter-lsp/jupyterlab-lsp/blob/v3.10.2/python_packages/jupyterlab_lsp/setup.cfg#L40)

Reply:
[f257678](https://github.com/conda-forge/jupyter-lsp-feedstock/pull/43/commits/f257678b96ee13de82d72a09bae241e959860259)